shared/locales/media/media.en-US.properties

```diff
@@ -0,0 +1,14 @@
+# Media
+
+byteUnit-B  = Bytes
+byteUnit-KB = KB
+byteUnit-MB = MB
+byteUnit-GB = GB
+byteUnit-TB = TB
+byteUnit-PB = PB
+byteUnit-EB = EB
+byteUnit-ZB = ZB
+byteUnit-YB = YB
+
+videoDescriptionShort = Video created at %I:%M %p, %B %e, %Y
+imageDescriptionShort = Image created at %I:%M %p, %B %e, %Y
```

Comment:
I just tested on iOS, and they don't use any verb at all. Simply "image, 12:45 pm, November 3rd 2012". That might be ambiguous, but at least it doesn't misinform.